### PR TITLE
Merge 19.01 into dev

### PR DESCRIPTION
**Pull request by nsoranzo**
@erasche scss files had merge conflicts, can you please check?

### CODE REVIEW

File: client/galaxy/style/scss/base.scss [hunk 1]
```diff
@@ -608,7 +608,7 @@ div.toolFormRow {
         @extend .py-1;
         @extend .px-2;
         cursor: move;
-        background: $panel-bg-color;
+        background: $base-color-3;
```
Comment (nsoranzo) [inline client/galaxy/style/scss/base.scss]: Mmh, this doesn't exist any more: https://travis-ci.org/galaxyproject/galaxy/jobs/492133195

**Comment by hexylena** [comment]
@nsoranzo scss LGTM

File: client/galaxy/style/scss/base.scss (hunk 1 above)
Comment (nsoranzo) [reply client/galaxy/style/scss/base.scss]: It was removed in https://github.com/galaxyproject/galaxy/pull/7203/files#diff-083a815db1387e189109dae6cce99198L7

**Comment by dannon** [comment]
In resolving the conflict (use of a now-nonexistent color var) there was a necessary change to the appearance of the workflow editor titles, seen here:  
image

They're not quite as dark as was introduced in #7309, but I think they still achieve the desired goal, and now they use a standard color.

File: client/galaxy/style/scss/base.scss (hunk 1 above)
Comment (hexylena) [reply client/galaxy/style/scss/base.scss]: Argh, only checked against my PR, and looked fine against that, didn't realise it was later removed. Sorry, should have rebuilt the client.

**Pull request merged**